### PR TITLE
BTPANELS - Add plugins commonly used by team panels

### PR DESCRIPTION
**Pull request by rjacoby**
Adds plugins:
- Markdown (with JavaFX view support)
- Bash
- Docker

Removes and ignores various Idea xml files that track frequency of feature usage.

### CODE REVIEW

File: config/codestyles/Default.xml [hunk 1]
```diff
@@ -0,0 +1 @@
+<code_scheme name="Default" version="173" />
```
Comment (rjacoby) [inline config/codestyles/Default.xml]: This seems to be an artifact of newer IDEA version.

Comment (pgr0ss) [reply config/codestyles/Default.xml]: You added it to the gitignore, but also included the file?

File: config/options/statistics.main_menu.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <component name="MainMenuCollector">
+    <counts>
+      <entry path="Build -&gt; Build Project" count="2" />
+      <entry path="Edit -&gt; Copy" count="1" />
+      <entry path="File -&gt; Exit" count="1" />
+      <entry path="File -&gt; Export Settings..." count="2" />
+      <entry path="File -&gt; Settings..." count="4" />
+    </counts>
+  </component>
+</application>
```
Comment (pgr0ss) [inline config/options/statistics.main_menu.xml]: Shouldn't all the statistics files be gone now that it's in the gitignore?

Comment (rjacoby) [reply config/options/statistics.main_menu.xml]: Yes; this one seems to have crept back in pull merge...

File: config/codestyles/Default.xml (hunk 1 above)
Comment (rjacoby) [reply config/codestyles/Default.xml]: This one I didn't ignore, we already had it checked in - do we want to remove it?

File: config/options/statistics.toolbar.clicks.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<application>
+  <component name="ToolbarClicksCollector">
+    <counts>
+      <entry action="com.intellij.build.CompositeView$SwitchViewAction@BuildView" count="2" />
+    </counts>
+  </component>
+</application>
```
[the remark this answers is not in the document]
Comment (rjacoby) [reply config/options/statistics.toolbar.clicks.xml]: Removed.

**Pull request merged**